polyfactory/value_generators/constrained_dates.py

```diff
@@ -7,11 +7,11 @@
 
 def handle_constrained_date(
     faker: "Faker",
-    ge: Optional[date] = None,
-    gt: Optional[date] = None,
-    le: Optional[date] = None,
-    lt: Optional[date] = None,
-) -> date:
+    ge: Optional[datetime] = None,
+    gt: Optional[datetime] = None,
+    le: Optional[datetime] = None,
+    lt: Optional[datetime] = None,
+) -> datetime:
```

Comment:
Y?

Reply:
@Goldziher this was done through ruff autofix, what do you recommend?